### PR TITLE
Jfinnerty/doc update

### PR DESCRIPTION
**Pull request by JJFinnerty**
Updates to WorkFlow Spec

### CODE REVIEW

File: guidelines/mwm-workflow-spec.md [hunk 1]
```diff
@@ -240,6 +244,7 @@ Example:
 ```python
 {{context.executions.body_part_identifier.result.body_part}} == 'leg'
 ```
+
 ##### DICOM Tags
 When the input data is DICOM, Evaluators can use DICOM tag values in conditional statements.
 DICOM tags are available in `context.dicom`. The reference for that object is as follows:
```
Comment (joshliberty) [inline guidelines/mwm-workflow-spec.md:250]: This needs updating

**Review by joshliberty — CHANGES_REQUESTED** [review]
Added a comment regarding the context.dicom which should be changed to context.input.dicom.
In addition - export destinations also need to change - since exports are now tasks, the previous task determines whether to trigger an export, based on the conditions of the task destination - the export destination itself doesn't need the additional condition.

File: guidelines/mwm-workflow-spec.md (hunk 1 above)
Comment (JJFinnerty) [reply guidelines/mwm-workflow-spec.md]: I've made the required updates.

**Comment by JJFinnerty** [comment]
Branch Highly Contminated with Changes I did not make.

**Pull request closed without merging**